bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/i18n/ThingStatusInfoI18nLocalizationService.java

```diff
@@ -15,6 +15,8 @@
 import java.util.Arrays;
 import java.util.Locale;
 
+import org.eclipse.jdt.annotation.NonNull;
```

Comment:
We agreed to not use `@NonNull`. Please use `@NonNullByDefault` for the whole file, or if not feasible for this PR, because it would involve too many changes in depending classes, just remove this annotation for now.

Reply:
OK; I will remove the annotations, as surrounding classes are not annotated yet.